diff --git a/user.js b/user.js
index aaddac3..f74bcae 100644
--- a/user.js
+++ b/user.js
@@ -850,7 +850,7 @@ user_pref("network.http.referer.hideOnionSource", true);
 /* 1610: ALL: enable the DNT (Do Not Track) HTTP header
  * [NOTE] DNT is enforced with Enhanced Tracking Protection regardless of this pref
  * [SETTING] Privacy & Security>Enhanced Tracking Protection>Send websites a "Do Not Track" signal... ***/
-user_pref("privacy.donottrackheader.enabled", true);
+user_pref("privacy.donottrackheader.enabled", false);
 
 /*** [SECTION 1700]: CONTAINERS
      If you want to *really* leverage containers, we highly recommend Temporary Containers [2].
