diff --git user.js user.js
index b61dc57..3bc266b 100644
--- user.js
+++ user.js
@@ -945,7 +945,7 @@ user_pref("media.getusermedia.audiocapture.enabled", false);
  * 0=Allow all, 1=Block non-muted media (default in FF67+), 2=Prompt (removed in FF66), 5=Block all (FF69+)
  * [NOTE] You can set exceptions under site permissions
  * [SETTING] Privacy & Security>Permissions>Autoplay>Settings>Default for all websites ***/
-   // user_pref("media.autoplay.default", 5);
+user_pref("media.autoplay.default", 5);
 /* 2031: disable autoplay of HTML5 media if you interacted with the site [FF78+]
  * 0=sticky (default), 1=transient, 2=user
  * [NOTE] If you have trouble with some video sites, then add an exception (see 2030)
