diff --git a/user.js b/user.js
index 9f74ed7..d9d74f1 100644
--- a/user.js
+++ b/user.js
@@ -1311,14 +1311,15 @@ user_pref("privacy.clearOnShutdown.siteSettings", false); // Site Preferences
  * Firefox remembers your last choices. This will reset them when you start Firefox.
  * [NOTE] Regardless of what you set privacy.cpd.downloads to, as soon as the dialog
  * for "Clear Recent History" is opened, it is synced to the same as 'history' ***/
-user_pref("privacy.cpd.cache", true);
-user_pref("privacy.cpd.cookies", true);
-   // user_pref("privacy.cpd.downloads", true); // not used, see note above
-user_pref("privacy.cpd.formdata", true); // Form & Search History
-user_pref("privacy.cpd.history", true); // Browsing & Download History
+user_pref("privacy.cpd.cache", false);
+user_pref("privacy.cpd.cookies", false);
+user_pref("privacy.cpd.downloads", false); // not used, see note above
+user_pref("privacy.cpd.formdata", false); // Form & Search History
+user_pref("privacy.cpd.history", false); // Browsing & Download History
 user_pref("privacy.cpd.offlineApps", true); // Offline Website Data
-user_pref("privacy.cpd.passwords", false); // this is not listed
-user_pref("privacy.cpd.sessions", true); // Active Logins
+user_pref("privacy.cpd.openWindows", false);
+user_pref("privacy.cpd.passwords", true); // this is not listed
+user_pref("privacy.cpd.sessions", false); // Active Logins
 user_pref("privacy.cpd.siteSettings", false); // Site Preferences
 /* 2805: clear Session Restore data when sanitizing on shutdown or manually [FF34+]
  * [NOTE] Not needed if Session Restore is not used (see 0102) or is already cleared with history (see 2803)
