diff --git a/user.js b/user.js
index 9f74ed7..3beba7b 100644
--- a/user.js
+++ b/user.js
@@ -603,7 +603,7 @@ user_pref("browser.cache.disk.enable", false);
  * extra session data contains contents of forms, scrollbar positions, cookies and POST data
  * define on which sites to save extra session data:
  * 0=everywhere, 1=unencrypted sites, 2=nowhere ***/
-user_pref("browser.sessionstore.privacy_level", 2);
+user_pref("browser.sessionstore.privacy_level", 0);
 /* 1022: disable resuming session from crash ***/
    // user_pref("browser.sessionstore.resume_from_crash", false);
 /* 1023: set the minimum interval between session save operations
